controllers/file.js

```diff
@@ -0,0 +1,93 @@
+'use strict';
+
+const express = require('express');
+const router = express.Router();
+const multer  = require('multer');
+const AWS = require('aws-sdk');
+const request = require('request');
+const moment = require('moment');
+const fs = require('fs');
+
+const upload = multer({ dest: 'uploads/' });
+const config = require('../config');
+
+AWS.config.update({
+  accessKeyId: config.aws.accessKeyId,
+  secretAccessKey: config.aws.secretAccessKey,
+  region: config.aws.region,
+  signatureVersion: config.aws.signatureVersion
+});
+
+const s3 = new AWS.S3();
+
+router.post('/', upload.single('document'), clamAV, s3Upload, deleteUpload, (req, res, next) => {
```

Comment:
personally I prefer functions to be declared before they are used, thought our lint kicked off if you define them after?